IPJ/IPJ6_PictureRGB256/src/main/java/Start.java

```diff
@@ -0,0 +1,17 @@
+/*
+Write a program that takes three integers between 0 and 255 that represent red, green and blue values of color
+and then creates and shows a 256-by-256 Picture of that color.
+Modified 3.1.2 from IPJ, page 362
+ */
+
+public class Start {
+    public static void main(String[] args) {
+        int[] RGB = new int[3];
+        RGB[0] = 123;
+        RGB[1] = 23;
+        RGB[2] = 3;
```

Comment:
1. You can inline this declaration (FYI)
2. It would be better to pass RGB values by args or by console
3. Dont' submit any Start class:) Write tests, to show how your program is working:)